hw/ip/aon_timer/doc/dv/index.md

```diff
@@ -85,22 +85,34 @@ Since the two timers are essentially independent, we use two test sequences, dri
 #### Test sequences
 
 The test sequences can be found in `hw/ip/aon_timer/dv/env/seq_lib`.
-The basic test virtual sequence configures the block in a valid initial state, but doesn't enable either timer.
+The basic test virtual sequence (aon_timer_base_vseq) configures the block in a valid initial state, but doesn't enable either timer.
 This is used as a base class for automated tests like the CSR tests.
 
-<div class="bd-callout bd-callout-warning">
+Smoke sequence initializes both timers, starts them with random threshold values configured.
+After that it waits until an interrupt is catched and clears everything before shutting down.
 
-**TODO**: Describe other test sequences here.
+Lock sequence tries to change the configuartion of the watchdog time while it is locked by software.
 
-</div>
+Stress sequence tries to change all configurations while running.
 
 #### Functional coverage
 
-<div class="bd-callout bd-callout-warning">
+To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
+The following cover points have been developed to prove that the test intent has been adequately met.
 
-**TODO**: Functional coverage points are not yet defined.
+`prescale_cp`: This cover point includes possible values for the prescale register of wakeup timer.
```

Comment:
Nit: I'd consider removing the "This cover point" prefix from each of these items.